src/factoid.coffee

```diff
@@ -23,9 +24,11 @@
 
 class Factoids
   constructor: (@robot) ->
-    @robot.brain.on 'loaded', =>
-      @cache = @robot.brain.data.factoids
-      @cache = {} unless @cache
+    storageLoaded = =>
+      @cache = @robot.brain.data.factoids ||= {}
+      @robot.logger.debug "Factoids Data Loaded: " + JSON.stringify(@cache, null, 2)
+    @robot.brain.on 'loaded', storageLoaded
+    storageLoaded() # just in case storage was loaded before we got here
```

Comment:
mmmm It was giving me problems with cache being always empty. I guess it is because the redis brain is  loaded before this plugin is loaded. I took the code from the plusplus plugin. 
https://github.com/ajacksified/hubot-plusplus/blob/master/src/scorekeeper.coffee
Maybe I can look into the history of that to see why it was changed. Sorry this are my first lines of coffeescript.